src/index.rs

```diff
@@ -17,20 +17,30 @@ use crate::Secp256k1Signer;
 #[cfg(feature = "cosmos")]
 use crate::CosmosSigner;
 
+#[cfg(feature = "aptos")]
+use crate::AptosSigner;
+
+#[cfg(feature = "aptos")]
+use crate::MultiAptosSigner;
+
 use crate::error::BundlrError;
 
 #[derive(FromPrimitive, Display, PartialEq, Eq, Debug, Clone)]
 pub enum SignerMap {
     None = -1,
     Arweave = 1,
-    Ed25519 = 2,
-    Secp256k1 = 3,
-    Cosmos = 4,
+    ED25519 = 2,
+    Ethereum = 3,
+    Solana = 4,
+    InjectedAptos = 5,
+    MultiAptos = 6,
+    Cosmos, //TODO: assign constant
 }
 
 pub struct Config {
     pub sig_length: usize,
     pub pub_length: usize,
+    pub sig_name: String,
```

Comment:
Rather store the signature type here and then implement `as_str` and maybe you want also `to_string` (that would just call `as_str().to_string()`).

Reply:
I'll mark this as future refactor

Reply:
These errors will need some further refactoring, also.